Inventory.cs

```diff
@@ -1,13 +1,14 @@
 ﻿using System;
 using InventoryManagementSystem.ProductInfo;
+using InventorySystem;
 
 namespace InventoryManagementSystem.InventoryInfo
 {
     public class Inventory
     {
         public string InventoryName { get; set; }
         private List<Product> _products = new();
-
+        private Connection _connection = new();
```

Comment:
We shouldn't be exposing the sql connection in our services, we should have a separate class to deal with the DB calls, and here we just call that class. So I would move all these queries and db calls to ProductRepository, and just use the repo here.

Reply:
and please check my comment here for Nayef, it will be helpful here as well 
https://github.com/Mohammad-Nayef/Inventory-Management-System/pull/7#discussion_r1382187437